app/presenters/field_of_operation_presenter.rb

```diff
@@ -17,16 +17,24 @@ def organisation
   end
 
   def description
-    description = @content_item["description"]
+    description = @content_item.description
 
     description.html_safe if description.present?
   end
 
   def contents
     contents = []
     contents << { href: "#field-of-operation", text: "Field of operation" } if description.present?
-    contents << { href: "#fatalities", text: "Fatalities" } if fatality_notices.present?
+    contents << { href: "#fatalities", text: "Fatalities" } if @content_item.fatality_notices.present?
 
     contents
   end
+
+  def roll_call_introduction(fatality_notice)
```

Comment:
I think we can merge commit [Add new tests for roll_call_introduction and casualties presenter methods](https://github.com/alphagov/frontend/pull/4742/commits/b7349acae639c9df578c0fc019d53a378c2b3f88) into this commit [Modify presenter and view to render content](https://github.com/alphagov/frontend/pull/4742/commits/54e14349e9bf78123c34d7a5cfe0cd5cece82f06)